libraries/botbuilder-ai-orchestrator/src/orchestratorComponentRegistration.ts

```diff
@@ -7,29 +7,20 @@
  */
 
 import { BoolExpressionConverter, NumberExpressionConverter, StringExpressionConverter } from 'adaptive-expressions';
-import { AdaptiveTypeBuilder } from 'botbuilder-dialogs-adaptive';
-import { BuilderRegistration, ComponentRegistration, ResourceExplorer } from 'botbuilder-dialogs-declarative';
 
 import { OrchestratorAdaptiveRecognizer } from './orchestratorAdaptiveRecognizer';
 
-export class OrchestratorComponentRegistration implements ComponentRegistration {
-    private readonly _builderRegistrations: BuilderRegistration[] = [];
-    private _resourceExplorer: ResourceExplorer;
-
-    public getTypeBuilders(): BuilderRegistration[] {
-        return this._builderRegistrations;
-    }
-
-    public constructor(resourceExplorer: ResourceExplorer) {
-        this._resourceExplorer = resourceExplorer;
-
-        this._builderRegistrations.push(
-            new BuilderRegistration('Microsoft.OrchestratorRecognizer', new AdaptiveTypeBuilder(OrchestratorAdaptiveRecognizer, this._resourceExplorer, {
+export class OrchestratorComponentRegistration {
+    public getDeclarativeTypes(_resourceExplorer: any) {
```

Comment:
We don't want to introduce dependency to `botbuilder-dialogs-declarative` into `botbuilder-ai-orchestrator`, so I have to use `any` instead of `ResourceExplorer` here.